Source/Renderer/Texture.js

```diff
@@ -148,6 +148,7 @@ define([
         this._preMultiplyAlpha = preMultiplyAlpha;
         this._flipY = flipY;
         this._sampler = undefined;
+        this.isDefault = false;
```

Comment:
I did it this way to avoid everyone having to handle it differently, but since Material is the only thing using defaultTexture right now, yes we can handle it all in Material.